.github/workflows/ci_windows.yaml

```diff
@@ -121,7 +121,7 @@ jobs:
             ${{ env.libtorrent_path }}
           cd ${{ env.libtorrent_path }}
           $env:CXXFLAGS+=" /guard:cf"
-          $env:LDFLAGS+=" /guard:cf"
+          $env:LDFLAGS+=" /GUARD:CF"
```

Comment:
Could we remove it from here? Isn't it enough that it is set in CMake configuration?

Reply:
>Isn't it enough that it is set in CMake configuration?

libtorrent don't have that flag iirc.